lib/evm_watchdog.rb

```diff
@@ -1,95 +0,0 @@
-require 'time'
-require 'linux_admin'
-require 'fileutils'
-
-module EvmWatchdog
-  PID_LOCATION = '/var/www/miq/vmdb/tmp/pids/evm.pid'.freeze
-  UPDATE_FILE  = '/var/www/miq/vmdb/tmp/miq_update'.freeze
-
-  def self.check_evm
```

Comment:
I feel like this method had other uses besides handling updates. It was rewritten in ruby in  

```
commit c9fcb1c345cd999f427c53dd2a035ebbe7b32b6f
Author: Brandon Dunne <bdunne@redhat.com>
Date:   Wed Nov 21 17:38:15 2012 +0000

    Base Watchdog functionality (including fix for empty file) re-written in ruby.

    BugzID: 16605
```

For example, the autostarting of the server process as soon as the db became available was one of the features that looks to still be here and was added in this commit:

```
commit 0dbf18b8b9c71e443aa15ee1725ea070fcb98e57
Author: Brandon Dunne <bdunne@redhat.com>
Date:   Thu Apr 26 15:36:10 2012 +0000

    Added support for restarting evm server when a database connection comes alive if server attempted to start with no database connection.
    Modified evm_watchdog accordingly.
    Removed unneeded RVM setup in evminit.
    Added support for testing database connection and getting server state from DB to evm_application.

    BugzID:14382
```

Reply:
Maybe @bdunne has opinions.  I don't remember if the original use cases for detecting db being down and coming back are still relevant.

Reply:
@kbrock This comment is no longer relevant with your new approach, right?